src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php

```diff
@@ -133,7 +133,9 @@ public function enrichDocument(
             ?: $base->getString('title');
         $doc->sp_sortvalue = $sortHeadline;
 
-        $doc->sp_startletter = $base->getString('startletter');
+        if ($base->has('startletter')) {
```

Comment:
Why is this necessary, `getString()` returns an empty string if the field does not exist.

Reply:
Bacause the solr Fiter 'sp_startletter:*' doesnt work
![solr](https://github.com/user-attachments/assets/44753b0e-74cb-4700-8e4d-e439954212fe)